### ISSUE TITLE
Investigation of systemic granulomatosis in cultured meagre, Argyrosomus regius, using clinical metagenomics

### ISSUE THREAD

**Issue by SvetlanaUP** [waldronlab/BugSigDBcuration#128]
Investigation of systemic granulomatosis in cultured meagre, Argyrosomus regius, using clinical metagenomics – Christina Pavloudi – Aquaculture
https://www.sciencedirect.com/science/article/pii/S0044848623000224
[ Pavloudi et al.pdf](https://github.com/waldronlab/BugSigDBcuration/files/12914249/Pavloudi.et.al.pdf)

**Comment by rmiyoyo** [comment]
## Name
- Raphael Miyoyo

Kindly requesting to be assigned to this issue.
@SvetlanaUP @cmirzayi

**Comment by SvetlanaUP** [comment]
Assigned.

**Comment by JigzG** [comment]
Can I curate this ?

**Comment by SvetlanaUP** [comment]
@rmiyoyo please let us know if you still want to curate this even though it's no longer during the Outreachy contribution period.

**Comment by BarakatAA** [comment]
@SvetlanaUP @Peacesandy  

Please can this be assigned to me?

**Comment by BarakatAA** [comment]
@imalovee @Scholarpat, @MyleeeA here is my paper, thank you

**Comment by BarakatAA** [comment]
@AleruDivine

**Comment by imalovee** [comment]
Alright sis

**Comment by BarakatAA** [comment]
@SvetlanaUP Please I just saw the needs review tag on my work. I'm not done yet and only tagged fellow contributors on slack as you asked us to for a peer review  :) I'll be sure to tag you once my work is ready

**Comment by BarakatAA** [comment]
@RotimiDesire

**Comment by BarakatAA** [comment]
Hi @SvetlanaUP  good morning! My paper is ready for review now 

https://bugsigdb.org/Study_817

**Comment by SvetlanaUP** [comment]
| Item | Description | max | points |
|---|---|---|---|
| 1 | All elements marked "Needs review" (none "Incomplete") | 1 | 1 |
| 2 | Correct study design | 1 | 1 |
| 3 | Entered all relevant experiments and no irrelevant experiments | 1 | 0 .5|
| 4 | Body site correctly identified (i.e. does not include multiple sites) | 1 | 1 |
| 5 | Condition entered according to contrast (correct EFO ontology) | 1 | 1 |
| 6 | Contrast groups correctly identified | 1 | 0.5 |
| 7 | Groups correctly labeled as 1 and 0 (1=cases, 0=controls) | 1 | 1 |
| 8 | Antibiotic exclusion correctly identified | 1 | 1 |
| 9 | Correctly identified sequencing details | 2 | 2 |
| 10 | Identified correct statistical test | 1 | 1 |
| 11 | Identified MHT correction | 1 | 1 |
| 12 | Correctly recorded matched on factors | 1 | 1 |
| 13 | Entered correct number of statistical tests per experiment | 1 | 1 |
| 14 | All diversity measures identified | 1 | 1 |
| 15 | Diversity results correctly entered as increased/decreased/unchanged | 1 | 1 |
| 16 | All signature sources correctly identified (-1 for each error) | 2 | 0.5 |
| 17 | Abundance direction correctly selected | 1 | 0.5 |
| 18 | Members of signatures identified correctly | 2 | 1 |
| 19 | Correct use of NCBI taxonomy | 2 | 1 |
|  | TOTAL | 23 | 18 |

This curation has 1 main error: instead of "Literature text 3.2", the curator should have used for curation figures available in the **Supplementary material**.
Minor comment: it's hard to follow this curation due to poor description and/or not providing detailed information

![FigS1](https://github.com/waldronlab/BugSigDBcuration/assets/66676084/37529aa6-2be0-4466-89d2-d0788ed4faf2)
![FigS2](https://github.com/waldronlab/BugSigDBcuration/assets/66676084/c8998f6c-1854-481c-9b9f-1370bb884ea9)
![FigS3](https://github.com/waldronlab/BugSigDBcuration/assets/66676084/066ffe20-52f2-42c4-844f-5994ba19dc0f)

**Comment by BarakatAA** [comment]
Thank you @SvetlanaUP  for the feedback and review!

I did not have access to the supplementary materials. When I requested for them on Slack, you provided the zipped folder attached below which did not include the supplementary figures hence why I referenced the Literature review text. I will now correctly use the supplementary figures for curation. Please share with me any other supplementary materials on this study you have access to, thank you.

[ScienceDirect_files_07Mar2024_16-31-28.335.zip](https://github.com/waldronlab/BugSigDBcuration/files/14874223/ScienceDirect_files_07Mar2024_16-31-28.335.zip)

Also, I opted for a cross sectional observational study design because the subjects used in this study (meagre fish) were not selected based on the presence or absence of a condition but were rather reared in a tank facility using borehole seawater under controlled conditions, euthanized and their kidney samples studied before classification into 3 groups based on the presence or absence of granuloma and calcification. The data and basis for this classification was obtained at a single point in time when the fishes were euthanized hence why I went with cross sectional observational design. However, I now understand my interpretation may have been wrong.

Thank you for your feedback and I’ll be sure to effect all corrections

**Comment by SvetlanaUP** [comment]
@BarakatAA the supplementary figures (Fig. S1-3) mentioned in the text are attached in my feedback. Feel free to do all the corrections and do let me know, thanks!

**Comment by BarakatAA** [comment]
Thank you @SvetlanaUP

**Comment by BarakatAA** [comment]
Good morning @SvetlanaUP , I've made the corrections to this study

Corrections made include:
Changing the study design from cross-sectional observational to case-control
Giving a more detailed and descriptive information for all curations and;
Using the Supplementary Figures (S1-S3) as curation source.

Thank you and enjoy the rest of your day

**Comment by SvetlanaUP** [comment]
https://bugsigdb.org/Study_817 reviewed.
